interfaces/python/CMakeLists.txt

```diff
@@ -277,17 +290,7 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/helics.py DESTINATION python COMPONENT
 if(WIN32)
     # Realistically for bundling into a Python wheel, the dll should be given a unique name to avoid
     # conflicts with other packages using a different version of the dll with the same name
-    add_custom_command(
```

Comment:
I think the copy for helicsSharedLib (+ libzmq if available) should remain for Windows builds.

Reply:
do you think we need to do the same for the other interface builds as well?

Reply:
Probably, because Windows doesn't have rpaths for finding libraries in other directories.

Reply:
Was hoping we could get rid of that but I guess not.  May have more tweaks when we start to test the matlab interface